File: Objc/Beacon/Beacon/BeaconInfo.m
//
//  BeaconInfo.m
//  Beacon
//
//  Created by Daher Alfawares on 12/9/14.
//  Copyright (c) 2014 Daher Alfawares. All rights reserved.
//

#import "BeaconInfo.h"

@implementation BeaconInfo

-(instancetype)initWithName:(NSString*)name withImageName:(NSString*)imageName withUUIDString:(NSString*)UUIDString withMajor:(NSInteger)major withMinor:(NSInteger)minor
{
    self = [super init];
    if(self)
    {
        _name  = name;
        _image = imageName;
        _uuid  = UUIDString;
        _major = major;
        _minor = minor;
    }
    return self;
}

@end

File: Beacon/Beacon/BeaconInfo.swift
//
//  BeaconInfo.swift
//  Beacon
//
//  Created by Daher Alfawares on 1/19/15.
//  Copyright (c) 2015 Daher Alfawares. All rights reserved.
//


/// Responsible for holding beacon data objects.

class BeaconInfo {
    
    var name : String
    var image : String
    var uuid : String
    var major : Int
    var minor : Int
    

    init( name : String, image : String, uuid : String, major: Int, minor: Int)
    {
        self.name = name
        self.image = image
        self.uuid = uuid
        self.major = major
        self.minor = minor
    }
}
